com.unity.mobile.notifications/CHANGELOG.md

```diff
@@ -2,10 +2,20 @@
 
 All notable changes to this package will be documented in this file.
 
+## [1.2.0-preview] - 2020-04-16
+
+### Changes & Improvements:
+- [Editor] Improved the Notification Settings UI, also stored the notification setting asset under 'Project Settings' folder rather than 'Assets' folder.
```

Comment:
It's 'ProjectSettings', not 'Project Settings' :)

Reply:
Nice catch. Fixed.